lucene/analysis/morfologik/ivy.xml

```diff
@@ -25,6 +25,7 @@
     <dependency org="org.carrot2" name="morfologik-polish" rev="${/org.carrot2/morfologik-polish}" conf="compile"/>
     <dependency org="org.carrot2" name="morfologik-fsa" rev="${/org.carrot2/morfologik-fsa}" conf="compile"/>
     <dependency org="org.carrot2" name="morfologik-stemming" rev="${/org.carrot2/morfologik-stemming}" conf="compile"/>
+    <dependency org="ua.net.nlp" name="morfologik-ukrainian-search" rev="${ua.net.nlp.morfologik-ukrainian-search.version}" conf="compile"/>
```

Comment:
The `rev` attribute value has to be of the form `"${/org/name}"` - this will not pass precommit.  You have to make such an entry in `ivy-versions.properties` - look at all the other examples there.